patches/tModLoader/Terraria/ModLoader/Logging.cs

```diff
@@ -77,7 +81,8 @@ internal static void LogStartup(bool dedServ)
 			tML.InfoFormat("Parsed Launch Parameters: {0}", string.Join(' ', Program.LaunchParameters.Select(p => ($"{p.Key} {p.Value}").Trim())));
 		}
 
-		DumpEnvVars();
+		if (!Program.LaunchParameters.ContainsKey("-build"))
```

Comment:
Why do we run `LogStartup` during build at all? What's the point if appenders haven't been configured?

Reply:
Thought that this at least produces stdout, but no. Will short-circuit.